tests/sentry/models/test_project.py

```diff
@@ -79,6 +109,20 @@ def test_transfer_to_organization(self):
         assert updated_rule.environment_id != rule.environment_id
         assert updated_rule.environment_id == Environment.get_or_create(project, "production").id
 
+        # check to make sure old monitor is scheduled for deletion
+        if SiloMode.get_current_mode() != SiloMode.REGION:
+            assert ScheduledDeletion.objects.count() == 1
```

Comment:
is this allowed @wedamija ? it was throwing an error otherwise. I couldn't find any documentation on it

Reply:
The silo mode stuff? Idk tbh, I'd ask hybrid cloud folks

Reply:
All good got it sorted

Reply:
`RegionScheduledDelete`